data/templates/install-coreos.sh

```diff
@@ -3,5 +3,11 @@ set -e
 CLOUD_CONFIG_FILE=pxe-cloud-config.yml
 curl -o $CLOUD_CONFIG_FILE http://<%=server%>:<%=port%>/api/current/templates/$CLOUD_CONFIG_FILE?nodeId=<%=nodeId%>
 sudo coreos-install -d <%=installDisk%> -c $CLOUD_CONFIG_FILE -b <%=repo%>
+
+# Customizations for supporting CoreOS Ignition:
+mkdir /mnt/coreos
+mount <%=installDisk%>6 /mnt/coreos/
```

Comment:
@tgelter : Thanks for you explanation, surely a strange writing here. Please at least add some comment here to make sure other people will not have my same feeling of typo.

Meanwhile, whether the number 6 partition table is specifically for your own system? Because as a general solution, I'd suggest adding one more parameter (such as `"partitionTableNumber"`) so it fits to more user cases.